### PR TITLE
Add macro C10_AS_INTARRAYREF_SLOW

### PR DESCRIPTION
**Pull request by ezyang**
This makes it easier to narrow down who is throwing the error,
instead of having to use gdb.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>
Differential Revision: [D42088781](https://our.internmc.facebook.com/intern/diff/D42088781)

### CODE REVIEW

File: .github/ci_commit_pins/xla.txt [hunk 1]
```diff
@@ -1 +1 @@
-06cc6d4f596e764502e3ce5ab998c8986fd52e5b
```
Comment (ngimel) [inline .github/ci_commit_pins/xla.txt:1]: accidental submodule update?

**Review by ngimel — APPROVED** [review]

**Review by malfet — APPROVED** [review]
LGTM, but please revert submodule update

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Comment by osalpekar** [comment]
Seems to have broken an internal call in `torch/fb/acc_tensor.cpp`:
```
[2022-12-14T16:40:43.212-08:00] Stderr: fbcode/caffe2/torch/fb/acc_runtime/acc_tensor.cpp:661:42: error: too few arguments to function call, expected 3, have 1
[CONTEXT] [2022-12-14T16:40:43.212-08:00]   return empty(at::asIntArrayRefSlow(size), dtype, device_ordinal);
[CONTEXT] [2022-12-14T16:40:43.212-08:00]   
```
See [D42052101](https://www.internalfb.com/diff/D42052101) for details.

**Comment by malfet** [comment]
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D42088781).

**Comment by malfet** [comment]
Merged directly as ShitIt is still broken

**Pull request merged**